src/pretix/control/templates/pretixcontrol/event/payment.html

```diff
@@ -48,19 +48,19 @@ <h1>{% trans "Payment settings" %}</h1>
                                 </a>
                             </td>
                         </tr>
-                    {% empty %}
+                    {% endfor %}
                         <tr>
-                            <td colspan="3">
+                            <td colspan="4">
+                                <br>
```

Comment:
To me, this break looks a little weird

```suggestion
                                <br>
```

Reply:
My objection was more with the "too much space" than the br as a tool, but that's a matter of taste then probably